### PR TITLE
Update events

### PR DESCRIPTION
**Pull request by cgebken**
THIS PR INCLUDES CHANGES FOR https://epages.atlassian.net/browse/UC-2186 (@Raigen) and https://epages.atlassian.net/browse/UC-3025 (@HgSlv). Please review the part related to the respective task. Thank you :)

### CODE REVIEW

File: apps/events.md [hunk 1]
```diff
@@ -153,23 +234,62 @@ checkoutState: null
 checkoutUrl: "https://pm.epages.com/epages/apidocu.sf/?ObjectPath=/Shops/apidocu/AnonymousUsers/1/Baskets/46072&ChangeAction=PickupBasket&PickupToken=MjNiZThlOGM0MjAzZWQ0N2ZjYmZmZDFiZjI3OTQxMzkwOWY4ZjZlOGE4NTM0ZWIxMjg3NmY2NzBiY2IxYzQ1OV8xNDgyMjIzMDcz"
 coupon: null
 couponCampaign: null
+customerEmail: null
 grandAmount: "76,01 €"
 grandAmountNote: "components.productComponent.priceExclusiveVat"
+grandTotal: {amount: 76.01, currency: "EUR", formatted: "€76,01"}
```
Comment (cgebken) [inline apps/events.md:240]: Please note that this example does not include the tax type (as stated in https://epages.atlassian.net/browse/UC-3025) as I didn't get that part when trying out the event myself.

File: apps/events.md [hunk 2]
```diff
@@ -14,9 +14,11 @@ These events are available:
 | Event | Type | Description |
 | - | - |  - |
 | `pageview` | string | Is triggered when the customer opens or reloads a page. Informs about the path of the page. |
-| `product` | immutable.js object | Is triggered when the customer selects a product. Informs about the respective product. |
+| `product` | immutable.js object | Is triggered when the customer selects a product. Informs about the respective product as well as the current state of the cart. |
 | `category` | immutable.js object | Is triggered when the customer selects a category. Informs about the respective category and related products.|
+| `cart` | object | Is triggered when the customer accesses the cart. Informs about the current state of the cart, e.g. included items. |
```
Comment (cgebken) [inline apps/events.md:19]: Is `object` correct?

File: apps/events.md [hunk 3]
```diff
@@ -14,9 +14,11 @@ These events are available:
...
 | `cart:add` | object | Is triggered when the customer adds a product to the cart. Informs about the current state of the cart, e.g. included items. This event isn't triggered if the customer makes changes in the cart itself, e.g. changing the amount of an item. |
+| `cart:setQuantity` | object | Is triggered when the quantity of a product in the cart is changed. This also includes the removal of a product. Informs about the quantity change and the affected product. |
```
Comment (cgebken) [inline apps/events.md:21]: Is `object` correct?

File: apps/events.md (hunk 2 above)
Comment (HgSlv) [reply apps/events.md:19]: Yes, it is

Comment (HgSlv) [reply apps/events.md:19]: Sorry, I just noticed there is also `immutable.js object`.
- For `product` event: product and cart should be `immutable.js object`
- For `cart:add` and `cart` event: cart should be `object`

File: apps/events.md [hunk 4]
```diff
@@ -130,14 +166,14 @@ updatedAt: "2018-04-17T14:48:29.000Z"
...
+## Cart event
 
 To make use of this event, you need to add the following snippet to your code:
 
 ```js
 if (window.eComEventTarget) {
-  window.eComEventTarget.addEventListener('cart:add', function (event) {
-    console.log('new cart data:', event.detail.cart)
+  window.eComEventTarget.addEventListener('cart', function (event) {
+    console.log('cart:', event.detail)
```
Comment (HgSlv) [inline apps/events.md:176]:
```suggestion
    console.log('cart:', event.detail.cart)
```

**Review by HgSlv — CHANGES_REQUESTED** [review]

**Review by HgSlv — APPROVED** [review]

File: apps/events.md (hunk 3 above)
Comment (Raigen) [reply apps/events.md:21]: Yes

**Review by Raigen — APPROVED** [review]

**Pull request merged**